packages/aws-cdk-lib/aws-cloudwatch/lib/alarm.ts

```diff
@@ -206,12 +227,22 @@ export class Alarm extends AlarmBase {
       ...metricProps,
     });
 
+    if (isAnomalyDetection) {
+      // Ensure returnData is set to true for both metrics in anomaly detection
+      const cfnMetrics = alarm.metrics as any[];
+      if (cfnMetrics && cfnMetrics.length >= 2) {
+        cfnMetrics[0].returnData = true;
+        cfnMetrics[1].returnData = true;
+      }
+    }
```

Comment:
Where is `cfnMetrics` used?

Reply:
Only in this block. To make sure the `returnData` property is set to true. TBH, the documentation is not super clear about which one should have the property set to true.

Reply:
Can you maybe provide the documentation in the comment? This code seems weird and providing the doc in the comment for future reference would help.